packages/babel-core/src/transformation/file/file.ts

```diff
@@ -44,15 +44,15 @@ export type NodeLocation = {
 };
 
 export default class File {
-  _map: Map<any, any> = new Map();
-  opts: any;
-  declarations: any = {};
-  path: NodePath<t.Program> = null;
-  ast: any = {};
+  _map: Map<unknown, unknown> = new Map();
+  opts: { [key: string]: any };
```

Comment:
Is it `ValidatedOptions` from `packages/babel-core/src/config/validation/options.ts` returned by `packages/babel-core/src/transformation/normalize-file.ts`? Or should we come up with a new type `NormalizedOptions`?

Reply:
Since I'm not familiar enough with babel internals, I can't confirm its type.
Maybe you can take a look after the PR is merged, I didn't change some code that was explicitly defined as any.